recipes/outrigger/meta.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: outrigger
+  version: "1.0.0"
+
+source:
+  fn: outrigger-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/13/b9/02d6db3140448685a9651918e3c2831401983d8123dd2560320283abddcc/outrigger-1.0.0.tar.gz
+  md5: 6b7c5a8ae752f79c49e2e124e609a099
```

Comment:
Please specify the `sha256` hash instead. You should be able to grab it from:

https://pypi.org/project/outrigger/#files